pkgs/development/libraries/vapoursynth/default.nix

```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
     sha256 = "09fj4k75cksx1imivqfyr945swlr8k392kkdgzldwc4404qv82s6";
   };
 
-  nativeBuildInputs = [ pkgconfig autoreconfHook nasm ];
+  nativeBuildInputs = [ pkgconfig autoreconfHook nasm python3 makeWrapper ];
```

Comment:
You now have two separate `python3` executables in `$PATH` in the builder. Given the libraries included in the one a couple lines down, I'm assuming that one is actually used during the build process, not this one?

Does vapoursynth's Python module have dependencies on any non-standard modules?

Reply:
This python3 seems to be needed to get the toPythonPath in the wrapProgram call below to work.

> Does vapoursynth's Python module have dependencies on any non-standard modules?

No, the current ones are only used during the build.